common/config.go

```diff
@@ -59,24 +59,117 @@ type AdminConfig struct {
 }
 
 type DatabaseConfig struct {
-	EvmJsonRpcCache *ConnectorConfig `yaml:"evmJsonRpcCache" json:"evmJsonRpcCache"`
+	EvmJsonRpcCache *CacheConfig `yaml:"evmJsonRpcCache" json:"evmJsonRpcCache"`
+}
+
+type CacheConfig struct {
+	Connectors []*ConnectorConfig   `yaml:"connectors" json:"connectors" tstype:"types.ConnectorConfig[]"`
+	Policies   []*CachePolicyConfig `yaml:"policies" json:"policies"`
+}
+
+func (c *CacheConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	type rawCacheConfig CacheConfig
+	raw := rawCacheConfig{}
+
+	if err := unmarshal(&raw); err != nil || (raw.Connectors == nil && raw.Policies == nil) || (len(raw.Connectors) == 0 && len(raw.Policies) == 0) {
+		// Backward compatibility for old config format
+		var rawBackward ConnectorConfig
+		if err := unmarshal(&rawBackward); err != nil {
+			return err
+		}
+		rawBackward.Id = "default"
+		c.Connectors = []*ConnectorConfig{&rawBackward}
+		c.Policies = []*CachePolicyConfig{
+			{
+				Network:   "*",
+				Method:    "*",
+				Finality:  DataFinalityStateFinalized,
+				TTL:       0,
+				Connector: "default",
+			},
+		}
+		// PostgreSQL might not be very efficient for these short default TTLs
+		if rawBackward.Driver != DriverPostgreSQL {
```

Comment:
how about `DDB`?

Reply:
DDB with native TTL is efficient to be used by default. In PGs case we have to run periodic intervals so is better to let user opt-in knowing the cost.